parachain/pallets/ethereum-beacon-client/src/ssz.rs

```diff
@@ -40,7 +40,7 @@ pub struct SSZSigningData {
 }
 
 #[derive(Default, SimpleSerialize, Clone, Debug)]
-pub struct SSZExecutionPayload {
+pub struct SSZExecutionPayloadHeaderCapella {
```

Comment:
```suggestion
pub struct SSZExecutionPayloadHeader {
```

Reply:
@vgeddes @claravanstaden Though I remove the versioned suffix in this PR I'am still a little hesitate about the idea reuse same `SSZExecutionPayload` for multiple verions with some `Option` field. 

The main reason is that we use some ssz libraries like [fastssz](https://github.com/ferranbt/fastssz) in relayer and  [ssz-rs](https://github.com/ralexstokes/ssz-rs) in parachain to generate ssz-binding for encoding&merkelization.

Not sure these libraies will handle `Option` type correctly as expected. Though checked with ralexstokes,author of `ssz-rs` should be fine for it. But There is no `Option` type I would assume use `Pointer` type instead in golang. But [sszgen](https://github.com/Snowfork/snowbridge/blob/a0f32d4f1079510e769257bfef4d907798e3e04a/relayer/README.md?plain=1#L49) can not regonize it and will just fail.

so I'll prefer explicitly add exclusive versioned structures without the `Option` workaround, that is also the normal way most light client like [prysm](https://github.com/prysmaticlabs/prysm/blob/797cc360c77c650751499e375da2c96be9b32b69/api/client/builder/types.go#L467) does, even means some duplication introduced but more straightforward for me.


Anyway, it's not the scope of this PR, I've archived the previous [Impl](https://github.com/Snowfork/snowbridge/tree/ron/versioning-capella-archive), could be still worth to check when we gonna implement versioning support later.

Reply:
This makes sense Ron. Thanks for investigating this 🙌

Reply:
> So I'll prefer explicitly add exclusive versioned structures without the Option workaround, that is also the normal way most light client like [prysm](https://github.com/prysmaticlabs/prysm/blob/797cc360c77c650751499e375da2c96be9b32b69/api/client/builder/types.go#L467) does, even means some duplication introduced but more straightforward.

I agree! Thanks @yrong!